ads/aqua/modeldeployment/deployment.py

```
@@ -871,14 +871,21 @@ def list(self, **kwargs) -> List["AquaDeployment"]:
                 if (
                     "datasciencemodelgroup" in aqua_model_id
                     or model_deployment.model_deployment_configuration_details.deployment_type
-                    == "UNKNOWN_ENUM_VALUE"
+                    in ["UNKNOWN_ENUM_VALUE", "MODEL_GROUP", "SINGLE_MODEL_FLEX"]
                 ):
                     continue
-                results.append(
-                    AquaDeployment.from_oci_model_deployment(
-                        model_deployment, self.region
+                try:
+                    results.append(
+                        AquaDeployment.from_oci_model_deployment(
+                            model_deployment, self.region
+                        )
                     )
-                )
+                except Exception as e:
+                    logger.error(
```

should we raise error here instead of continuing with the deployment?